src/entries/popup/pages/swap/index.tsx

```diff
@@ -605,6 +596,37 @@ export function Swap({ bridge = false }: { bridge?: boolean }) {
     resetSwapValues,
   } = usePopupInstanceStore();
 
+  const bridgingSelected = useMemo(() => {
+    const assetToSellAddressToCompare =
+      assetToSell?.[
+        assetToSell?.chainId === ChainId.mainnet ? 'address' : 'mainnetAddress'
+      ];
+    const assetToBuyAddressToCompare =
+      assetToBuy?.[
+        assetToBuy?.chainId === ChainId.mainnet ? 'address' : 'mainnetAddress'
+      ];
+    return isLowerCaseMatch(
+      assetToSellAddressToCompare,
+      assetToBuyAddressToCompare,
+    );
+  }, [assetToBuy, assetToSell]);
```

Comment:
wdyt of `isSameAssetInDiffChains(assetToSell, assetToBuy);`?

Reply:
👍